modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/search/ExecutorUtil.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.search.elasticsearch7.internal.search.engine.adapter.search;
+
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.search.searcher.SearchTimeValue;
+
+import java.util.concurrent.TimeUnit;
+
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+
+/**
+ * @author Gustavo Lima
+ */
+public class ExecutorUtil {
+
+	protected static long getMinutes(SearchTimeValue searchTimeValue) {
+		TimeUnit timeUnit = searchTimeValue.getTimeUnit();
```

Comment:
do we need to handle if searchTimeValue == null?

Reply:
If the user does not set the searchTimeValue what should be the scroll time? 0? 1?